### ISSUE TITLE
kubeless-ui doesnt support functions in other namespaces

### ISSUE THREAD

**Issue by dimagoldin** [kubeless/kubeless-ui#70]
functions deployed in namespaces other then "default" are not shown.
also if the config: functions-namespace, is used, kubeless-ui does not show anything at all

**Comment by botzill** [comment]
Yes, it's related to this: https://github.com/kubeless/kubeless/issues/1015

**Comment by leavest** [comment]
![image](https://user-images.githubusercontent.com/30454841/68097004-65cbba80-feef-11e9-8a55-ec6026c3df30.png)
I create two functions in different namspace with the same name.
How can I distinguish them in ui?

**Comment by andresmgot** [comment]
unfortunately this UI is not being maintained. We are open to PR to fix or implement new features though.
